app/assets/stylesheets/application.scss

```diff
@@ -11,8 +11,9 @@
  * It is generally better to create a new file per style scope.
  *
  */
+
 @import 'font-awesome';
 @import 'bootstrap';
-@import 'layout/**/*';
 @import 'abstracts/_colors';
 @import 'abstracts/_buttons';
+@import 'layout/**/*';
```

Comment:
I don't see why you are importing files inside `layout/something/something` I think you can simply do `layout/*`

Reply:
Agree!